graf3d/eve7/src/REveTableInfo.cxx

```diff
@@ -78,6 +84,47 @@ void REveTableViewInfo::AddNewColumnToCurrentCollection(const std::string& expr,
 //  Look for definition also in base classes
 REveTableHandle::Entries_t& REveTableViewInfo::RefTableEntries(std::string cname)
 {
+   struct SimpleMethodFill
+   {
+      void fill(REveTableHandle::Entries_t& entries, TClass* c)
+      {
+         TMethod *meth;
+         TIter    next(c->GetListOfMethods());
+         while ((meth = (TMethod*) next())) {
+            // take only methods without arguments
+            if (!meth->GetListOfMethodArgs()->First())
+            {
+               std::string mn = meth->GetName();
+               std::string exp = "i." + mn + "()";
+
+               // ?? AMT is thre a better way to get detect numeric types
+               std::string r  = meth->GetReturnTypeName();
+               if ( r == "int"    || r == "Int_t" ||
```

Comment:
Alja, you can try to use `gROOT->GetDataType(name)` method. Return is `TDataType` class which can be used to test if it integer or double or other types you expects.

Reply:
Thank you for the suggestion.